sourced/ml/transformers/tfidf.py

```diff
@@ -0,0 +1,63 @@
+from collections import namedtuple
+import typing
+
+import pyspark
+
+from sourced.ml.algorithms import log_tf_log_idf
+from sourced.ml.transformers.transformer import Transformer
+
+TFIDF_Row = namedtuple("TDIDF_Row", ["token", "document", "value"])
```

Comment:
UpperCamelCase

Reply:
`TFIDFRow` or `TfIdfRow` or `TFIDFRow`?

Reply:
`TFIDFRow` - TFIDF is a common acronym

Reply:
ok, I decide that it is better to use just `pyspark.Row` everywhere :)